skeleton/nix/util.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs.lib;
+
+{
+  # TODO: Replace this with a function that identifies packages from your
+  # project.
+  # fixme: automate this in nix-tools/Haskell.nix.
+  isIohkSkeleton = package:
+    (hasPrefix "iohk-skeleton" package.identifier.name) ||
+    (elem package.identifier.name [ "another-package" ]);
```

Comment:
It is not clear to me whether I can remove the right operand of the `||`, namely:

```nix
(elem package.identifier.name [ "another-package" ])
```